File: src/main/java/de/eugenbernwald/fretboardtrainer/ui/MixerInfoStringConverter.java
package de.eugenbernwald.fretboardtrainer.ui;

import javafx.util.StringConverter;

import javax.sound.sampled.Mixer;

public class MixerInfoStringConverter extends StringConverter<Mixer.Info> {

    @Override
    public String toString(Mixer.Info info) {
        return info.getName();
    }

    @Override
    public Mixer.Info fromString(String string) {
        return null;
    }
}

File: src/main/java/de/eugenbernwald/fretboardtrainer/ui/MixerInfoStringConverter.kt
package de.eugenbernwald.fretboardtrainer.ui

import javafx.util.StringConverter

import javax.sound.sampled.Mixer

class MixerInfoStringConverter : StringConverter<Mixer.Info>() {

    override fun toString(info: Mixer.Info): String {
        return info.name
    }

    override fun fromString(string: String): Mixer.Info? {
        return null
    }
}
